README.md

```diff
@@ -165,6 +166,7 @@ helm upgrade --install leanix-k8s-connector leanix/leanix-k8s-connector \
 --set args.processingMode=full \
 --set args.lxWorkspace=00000000-0000-0000-0000-000000000000 \
 --set args.verbose=true \
+--set args.enableCustomStorage=true \
```

Comment:
@aravindmetku  Lets specifically indicate the example also covering the case when `enablecustomStorage=false` to indicate the `args` that are not needed to be supplied when the custom storage option is disabled.

Reply:
done